_posts/2021-08-09-fall-2021-meeting-announced.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "PyHC Fall 2021 Meeting Dates Announced."
+author: jibarnum
+---
+
+PyHC serves as a community knowledge base for performing heliophysics research in Python, aiming to provide a variety of tutorials and resources, a list of useful packages, general discussion, and advice. Twice yearly, PyHC meets in person. These meetings are open to anyone, from individuals already utilizing Python heavily in developing open-source heliophysics packages to those who are just getting into using Python and open source development. 
+
+**PyHC's second meeting of 2021 will be held remotely on Zoom. The meeting will run Monday, October 25th, 2021 - Thursday, October 28th, 2021, from 9 AM - 11 AM MT each day.** _The Zoom sessions will be recorded._ We’ll hold tutorial, hackathon, and project update sessions, as well as a couple other discussions that are relevant to the community. See [the full agenda](https://docs.google.com/spreadsheets/d/19zVGD72ZFPrKS-o2DYjrBVAHtEY8EKEkFGjYa21XQgU/edit?usp=sharing) for dates/times of the various sessions. Note that the agenda is subject to change. 
+
+The meeting's web page will be created shortly (and the link will be put here). **Registration, which is free, but required, can be found [here](https://forms.gle/dchiQzPHkcjf3SUQ7).** The meeting's web page also has a link to registration, as well as the Zoom telecon information. If you’d like to join our mailing list to learn more information about the meeting and other upcoming events, please see the [PyHC Contact page](http://heliopython.org/contact/) for instructions. We also have an [Element chat group]( https://riot.im/app/#/room/#heliopython:openastronomy.org
```

Comment:
Looks great! In this second sentence, there's no comma needed between "which is free" and "but required." So I'd remove that, but otherwise this is good to merge.

Reply:
Aye just realized I can edit this file myself. Yay not working off personal forks. Change made, gonna merge.